test/unit_test/main.cpp

```diff
@@ -0,0 +1,135 @@
+#include "../../src/traces/traces.hpp"
+#include <vector>
+#include <unordered_map>
+#include <iostream>
+#include "../../src/sequential/sequential.hpp"
+#include "../../src/parallel/parallel.hpp"
+#include <unordered_set>
+
+#include "../../src/common/types.hpp"
+
+
+std::unordered_map<KeyType, ValueType> map;
```

Comment:
declare globally or in function? in the perf test they were declared in test function